clarifai/datasets/upload/image.py

```diff
@@ -10,8 +10,8 @@
 
 class VisualClassificationDataset(ClarifaiDataset):
 
-  def __init__(self, datagen_object: Iterator, dataset_id: str, split: str) -> None:
-    super().__init__(datagen_object, dataset_id, split)
+  def __init__(self, datagen_object: Iterator, dataset_id: str) -> None:
```

Comment:
Same for all references.
```suggestion
  def __init__(self, data_generator: Iterator, dataset_id: str) -> None:
```